docs/reference/search/field-caps.asciidoc

```diff
@@ -53,13 +53,33 @@ include::{docdir}/rest-api/common-parms.asciidoc[tag=fields]
 include::{docdir}/rest-api/common-parms.asciidoc[tag=index-ignore-unavailable]
 
 `include_unmapped`::
-  (Optional, boolean) If `true`, unmapped fields are included in the response. 
+  (Optional, boolean) If `true`, unmapped fields are included in the response.
   Defaults to `false`.
 
+[[search-field-caps-api-request-body]]
+==== {api-request-body-title}
+
+`index_filter`::
+(Optional,  <<query-dsl,query object>> Allows to filter indices if the provided
+query rewrites to `match_none` on every shard.
++
+--
+[IMPORTANT]
```

Comment:
Looks like we've duplicated this whole note, maybe we could just keep the one below?